src/main/java/com/conveyal/datatools/manager/jobs/MergeStrategy.java

```diff
@@ -0,0 +1,46 @@
+package com.conveyal.datatools.manager.jobs;
+
+/**
+ * This enum defines the different strategies for merging, which is currently dependent on whether trip_ids and/or
+ * service_ids between the two feeds are exactly matching.
+ */
+public enum MergeStrategy {
```

Comment:
Here's another place where types are used instead of some kind of interface/abstract class with implementing/extending classes that perform the relevant behavior at the desired time. Furthermore, it's additionally confusing since there is already the `MergeFeedsType`. At the very least, since this only operates on the ServicePeriod merge type, maybe it can be called something a bit different like ServicePeriodMergeTypeResolutionStrategy?